src/sentry/tasks/unmerge.py

```diff
@@ -225,6 +226,10 @@ def migrate_events(
         group=destination_id
     )
 
+    EventAttachment.objects.filter(project_id=project.id, event_id__in=event_id_set).update(
```

Comment:
can filter ever return None ?

Reply:
I don't think so, it would be an empty queryset

Reply:
mind testing it before committing ?